DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/3RicercaDeiProdotti.tex

```diff
@@ -1,35 +1,31 @@
-\subsection{UC5 - Ricerca dei prodotti} \label{UC5}
-
-\begin{figure}[H]
-    \centering
-    \includegraphics[width=\textwidth]{Immagini/DiagrammiUC/UC5RicercaDeiProdotti.png}
-    \caption{Diagramma di UC5: Ricerca dei prodotti} 
-    \label{fig:RicercaDeiProdotti}
-\end{figure}
-
-L'utente non autenticato o l'acquirente può ricercare i prodotti dalla home oppure dalla \glo{PLP}.
+\subsection{UC5 - Ricerca dei prodotti} \label{UC5} 
+L'utente non autenticato o l'acquirente può ricercare i prodotti dalla schermata principale oppure dalla \glo{PLP}.
 \begin{itemize}
-    \item \textbf{Attori Primari:} Acquirente; Utente non autenticato.
-    \item \textbf{Precondizione:} L'attore ha selezionato l'azione della ricerca e inserito le parole per la ricerca del prodotto.
-    \item \textbf{Postcondizione:} L'attore viene spostato alla PLP con i prodotti che contengono almeno una delle parole ricercate nella descrizione o nel nome. 
-    \item \textbf{Scenario Principale:} 
+    \item \textbf{Attori primari:} acquirente o utente non autenticato;
+    \item \textbf{Precondizione:} l'attore ha selezionato l'azione della ricerca e inserito le parole per la ricerca del prodotto;
+    \item \textbf{Postcondizione:} l'attore viene spostato alla PLP con i prodotti che contengono almeno una delle parole ricercate nella descrizione o nel nome;
+    \item \textbf{Scenario principale:} 
     \begin{itemize}
-        \item L'attore inserisce le parole per la ricerca del prodotto.
-        \item Preme sull'azione di ricerca.
-        \item Viene portato alla PLP che contiene che contiene tutti i prodotti che hanno almeno una delle parole indicate nella descrizione o il titolo
+        \item L'attore inserisce le parole per la ricerca del prodotto;
+        \item Preme sull'azione di ricerca;
+        \item Viene portato alla PLP che contiene tutti i prodotti che hanno almeno una delle parole indicate nella descrizione o il titolo
+    \end{itemize}
+    \item \textbf{Estensioni:} l'utente ricerca dei prodotti non presenti nella piattaforma.
+    \begin{itemize}
+    	\item (UC5.1) - Visualizzazione messaggio di ricerca senza alcun risultato.
```

Comment:
Inserire scenario alternativo nel quale va spiegato quando avviene l'errore (guardare come è stato fatto nella gestione delle categorie del venditore)

Reply:
Gli scenari alternativi io e Cristiano li abbiamo tolti perchè nell’esempio del prof sulle sue slide c’era solo l’estensione con la riga di spiegazione che non mettevamo in scenario alternativo sempre dentro all’estensione

Reply:
Domani ne parliamo con tutti